packages/icons/icons.json

```
@@ -373,91 +373,91 @@
     {
         "displayName": "Import",
         "iconName": "import",
-        "tags": "arrow, down, importing,",
+        "tags": "arrow, down, importing, , download",
         "group": "action",
         "codepoint": 61946
     },
     {
         "displayName": "Export",
         "iconName": "export",
-        "tags": "arrow, up, exporting",
+        "tags": "arrow, up, exporting, output, move out, upload, send",
         "group": "action",
         "codepoint": 61856
     },
     {
         "displayName": "Minimize",
         "iconName": "minimize",
-        "tags": "arrows, decrease, smaller",
+        "tags": "arrows, decrease, smaller, combine, shrink, compress, together, crash, resize, collision, close, connect, merge, conflict",
         "group": "action",
         "codepoint": 62019
     },
     {
         "displayName": "Maximize",
         "iconName": "maximize",
-        "tags": "arrows, increase, bigger",
+        "tags": "arrows, increase, bigger, expand, opposite diagonal arrows, diagonal bidirectional arrows, double diagonal arrows",
         "group": "action",
         "codepoint": 62012
     },
     {
         "displayName": "Tick",
         "iconName": "tick",
-        "tags": "mark, sign, ok, approved, success",
+        "tags": "mark, sign, ok, approved, success, check, verizon, done, complete, confirm",
```

```suggestion
        "tags": "mark, sign, ok, approved, success, check, done, complete, confirm",
```

not a fan of the brand name addition, but the other ones are good